requirements.txt

```diff
@@ -9,4 +9,6 @@ requests>=2.22.0,<3
 label-studio-tools>=0.0.0.dev11
 Jinja2==3.0.3
 itsdangerous==2.0.1
-werkzeug==2.0.2
\ No newline at end of file
+werkzeug==2.0.2
+redis
```

Comment:
Could you please freeze the versions of these packages? 
better to use >= and <= e.g. 
rq>=1.2.3,<=4.5.6